docker/sp1/Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN chmod +x /tmp/install_sp1_sdk.sh
 # The install_sp1_sdk.sh script will respect these ENV variables.
 # TODO: we are hardcoding /root which may not work for other users
 ENV SP1UP_HOME="/root/.sp1up" \
+    SP1UP_VERSION="v5.0.8" \
```

Comment:
I changed the `install_sp1_sdk.sh` below to support this extra env flag.

If we don't do this, the SP1 docker image cache will become stale quite fast. Since "nothing change" future runs will use cached images since there's no way to distinguish if there's a new SP1 SDK as part of the build process.

Also is important to have reproducible builds. If we always download the "latest release", even with all this dockerization we can suddenly broken compilations if a new SDK release broke something (as happened some days ago both to me and @kevaundray)

Reply:
Sorry, I think I put the wrong name here. This is the SDK version that will be used by `sp1up`. I should probably rename it to `SP1_SDK_INSTALL_VERSION`.

Does that clarify the confusion? Or maybe your comment still applies?

Reply:
I renamed it.